Part of extended mode being an argument and not a compile time decision.
Some bits about using vfork, or not, and pidfile as an argument.

--- vtund.8.orig	2016-09-17 20:01:21 UTC
+++ vtund.8
@@ -48,6 +48,12 @@ vtund \- VTun(Virtual Tunnel) daemon.
 [ 
 .I -n 
 ] 
+[ 
+.I -e 
+] 
+[ 
+.I -s pid_file 
+] 
 <
 .I session 
 >
@@ -115,6 +121,14 @@ Connect 
 .I timeout
 Default is 30 seconds. This options is equivalent to the 'timeout' option of
 config file.
+.TP
+.I -e
+Use vtun-v6 extended mode.
+.TP
+.I -z pid_file
+Connect 
+.I z pid_file
+The full pathname of the file containing the PID of the daemon in server mode.
 .TP
 .I session 
 Session name from the config file.
